### PR TITLE
AccountsService: Add PrefersAccentColor to users

### LINKED ISSUE #175: Add accent color support to AccountsService
Similar to our prefer-color-scheme stuff, but to save the user's accent color so we can communicate that to the greeter, session indicator and maybe Flatpak apps eventually.

### PR DESCRIPTION
**Pull request by cassidyjames**
Will fix #175. Open to feedback.

My reason for having `0` equal "No Preference" while also having blue is because it seemed the most flexible if we wanted to change how the accent color stuff worked down the line, without hardcoding the default to blue. But maybe this is overthinking it.

### CODE REVIEW

**Comment by cassidyjames** [comment]
Currently unsure how we want to refer to the "generic" names, but mapping them to an int/enum means we can tweak them in the comment later without affecting anything code-wise—so probably best not to get too hung up on it.

File: accountsservice/io.elementary.pantheon.AccountsService.xml [hunk 1]
```diff
@@ -16,6 +16,21 @@
       <annotation name="org.freedesktop.Accounts.DefaultValue.String" value="24h"/>
     </property>
 
+    <!--
+      PrefersAccentColor:
+
+      What accent color the user prefers for the UI.
+
+      Valid preferences: 0 = No Preference (currently assumes Blue), 1 = Red, 
+      2 = Orange, 3 = Yellow, 4 = Green, 6 = Mint, 6 = Blue, 7 = Purple,
```
Comment (danrabbit) [inline accountsservice/io.elementary.pantheon.AccountsService.xml:25]: You have a typo here with 5, 6

**Review by danrabbit — CHANGES_REQUESTED** [review]
Noticed a typo. I wonder if it would be more legible to just have each value on its own line

**Comment by danrabbit** [comment]
@cassidyjames this seems reasonable to me, should this be marked ready for review?

**Review by danrabbit — APPROVED** [review]
:shipit:

**Pull request merged**